spoj_practice/spoj2/spoj2.py

```diff
@@ -51,12 +38,14 @@ def find_primes_between (n, m):
     # m = numbers[0]
     # n = numbers[1]
     # instead, do:
-    m, n = stdin.readline().split(' ')
-    m = int(m)
-    n = int(n)
+    m, n = map(int, stdin.readline().split(' '))
     prime = []
```

Comment:
1. In Python, the function just assigns an entire new list object to `primes` when you call it (specifically, it is the instance that was created inside the `find_primes_between` function). It doesn't go through the existing `primes` list and replace values at various indices. The old object is just dropped and the new one is asigned

2. Even if this was necessary, I think you wanted `primes = []` here, and not `prime = []`